### ISSUE TITLE
Include not working after 4.14.9 changes

### ISSUE THREAD

**Issue by ivankustera** [marko-js/marko#1234]
### Marko Version: 

^4.14.9

### Details

After upgrading to ^4.14.9 include stopped working, throws error instead. Tried even the `<${dynamic}/>` tag but failed to make it work.

### Expected Behavior

It should render the included template

### Actual Behavior

Error is thrown

``` 
my_included_marko_file.marko:1
(function (exports, require, module, __filename, __dirname) { <div class="page">
                                                              ^

SyntaxError: Unexpected token <
```

### Your Environment

- node v10.13.0
- macOS High Sierra v10.13.6

### Steps to Reproduce

Here's the main template that's being rendered:

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
    <style>
        ${input.cssContent}
    </style>
  </head>
  <body>
    <div class="container">
      <if(input.renderFullReport)>
        <include('./report-full.marko', input)/>
      </if>
      <else>
        <include('./report-individual.marko', input)/>
      </else>
    </div>
  </body>
</html>
```

`report-full.marko` looks like:

```
<div class="page">
  <div class="header">
    <div class="header-content">
        ...
```

`report-individual.marko` looks like:

```
$ var setId = function(input, value) {
    return (`individual_${input.specId[0]}_${value}`);
  }
<div class="page">
  <div class="header">
    <div class="header-content">
        ...
```

### Stack Trace

```
SyntaxError: Unexpected token <
      at new Script (vm.js:79:7)
      at createScript (vm.js:251:10)
      at Object.runInThisContext (vm.js:303:10)
      at Module._compile (internal/modules/cjs/loader.js:656:28)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
      at Module.load (internal/modules/cjs/loader.js:598:32)
      at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
      at Function.Module._load (internal/modules/cjs/loader.js:529:3)
      at Module.require (internal/modules/cjs/loader.js:636:17)
      at require (internal/modules/cjs/helpers.js:20:18)
      at Object.<anonymous> (specification_layout/report-template.marko.js:9:29)
      at Module._compile (internal/modules/cjs/loader.js:688:30)
      at loadSource (node_modules/marko/src/loader/index-default.js:73:20)
      at doLoad (node_modules/marko/src/loader/index-default.js:174:24)
      at Object.load (node_modules/marko/src/loader/index-default.js:37:16)
```

</details>

**Comment by DylanPiercey** [comment]
Thanks for creating the issue. I think I know what’s going on here and we can work to get a patch out soon.

In the mean time a workaround should be able to use the Marko require hook before loading any templates like so:

`require('marko/node-require').install();`

**Comment by ivankustera** [comment]
Thanks for the quick response. It's not so urgent so I can wait for the patch. Hopefully it's work with the `<${dynamic}/>` tag so I can do upgrade and the migration at the same time.

**Comment by DylanPiercey** [comment]
This is now fix and released as 4.14.22 😄.
